test/e2e/framework/constant.go

```diff
@@ -0,0 +1,10 @@
+package framework
+
+import "time"
+
+const (
+	// pollInterval defines the interval time for a poll operation.
+	pollInterval = 5 * time.Second
+	// pollTimeout defines the time after which the poll operation times out.
+	pollTimeout = 420 * time.Second
```

Comment:
420s is too long, e2e should close to real usage.

Reply:
ok

Reply:
420 is set with reference to Karamda.

Reply:
ok